crates/turborepo-ui/src/tui/pane.rs

```diff
@@ -77,13 +73,12 @@ impl<'a, W> Widget for &TerminalPane<'a, W> {
     {
         let screen = self.terminal_output.parser.screen();
         let block = Block::default()
-            .title(self.terminal_output.title(self.task_name))
-            .title_bottom(self.footer())
-            .style(if self.highlight() {
```

Comment:
We don't do these yellow highlights. The only place in the program we do yellow highlights now is in the task table (not here), so we can remove this.